submitter/relayer/relayer.go

```diff
@@ -386,44 +386,54 @@ func (rl *Relayer) buildTxWithData(
 	rl.logger.Debugf("Building a BTC tx using %v with data %x", utxo.TxID.String(), data)
 	tx := wire.NewMsgTx(wire.TxVersion)
 
-	outPoint := wire.NewOutPoint(utxo.TxID, utxo.Vout)
-	txIn := wire.NewTxIn(outPoint, nil, nil)
-	// Enable replace-by-fee
-	// See https://river.com/learn/terms/r/replace-by-fee-rbf
-	txIn.Sequence = math.MaxUint32 - 2
-	tx.AddTxIn(txIn)
-
-	// build txout for data
+	// build txOut for data
 	builder := txscript.NewScriptBuilder()
 	dataScript, err := builder.AddOp(txscript.OP_RETURN).AddData(data).Script()
 	if err != nil {
 		return nil, err
 	}
 	tx.AddTxOut(wire.NewTxOut(0, dataScript))
 
-	// build txout for change
-	changeAddr, err := rl.GetChangeAddress()
+	feeRate := btcutil.Amount(rl.getFeeRate()).ToBTC()
+	rawTxResult, err := rl.BTCWallet.FundRawTransaction(tx, btcjson.FundRawTransactionOpts{
```

Comment:
While looking at this function docs - https://developer.bitcoin.org/reference/rpc/fundrawtransaction.html#argument-2-options
it seems we can specify the position of change output in arguments - `changePosition`.

Maybe it would be better to use this option instead finding change output on our own ?